From 296c902be839b83eec34c6da21e79578cc1ae89a Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Mon, 25 Jun 2018 17:32:35 +0100
Subject: [PATCH] MANTA-3781 manta-mackerel build asks for tar when it wants
 gtar Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 00f283a..3ad09ea 100644
--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ publish: release
 
 .PHONY: assets
 assets: $(NODEUNIT)
-	tar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
+	gtar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
 
 
 include ./tools/mk/Makefile.deps
-- 
2.21.0

